pkg/kube/kube.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 ReactiveOps.
+Copyright 2019 ReactiveOps.
```

Comment:
I noticed you changed this in some places but not all, is there a particular reason for that?

Reply:
I think the goal was to set 2019 on new files and leave old files at 2018, but it looks like this is an exception to that. I don't _think_ the year matters very much here, but I'm far from an expert on this.